pom.xml

```diff
@@ -75,8 +77,8 @@
     </repositories>
 
     <build>
-        <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
-        <testSourceDirectory>${project.basedir}/src/test/java</testSourceDirectory>
+<!--        <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>-->
+<!--        <testSourceDirectory>${project.basedir}/src/test/java</testSourceDirectory>-->
```

Comment:
Don't comment this out?

Reply:
> I am not really sure what to say about this, thanks for the effort of course but I think you seemed to have trouble understanding this?
> If you need any help with the task, feel free to join our discord -> https;//discord.gg/slimefun
> and we will gladly help you.
> 
> The Unit Tests should cover the energy distribution algorithm, so you would have had to mock the EnergyNet class itself and provide example configurations to then test how the energy is distributed inbetween them.

Yeah. Bit more challenging. I will join the discord the next day.

Reply:
No worries! I've converted this into a draft for now, so that we can guide you through it before reviewing it.

Reply:
Thanks @TheBusyBiscuit ill be joining the discord server also so that we can resolve this.